packages/Webkul/API/Http/Resources/Catalog/Product.php

```diff
@@ -37,7 +37,7 @@ public function toArray($request)
             'name'                   => $this->name,
             'url_key'                => $this->url_key,
             'price'                  => $product->getTypeInstance()->getMinimalPrice(),
-            'formated_price'         => core()->currency($product->getTypeInstance()->getMinimalPrice()),
+            'formatted_price'         => core()->currency($product->getTypeInstance()->getMinimalPrice()),
```

Comment:
Keep in mind that this is a breaking change.

Live systems, that are using this api endpoint, will still use 'formated_price' and will fail after this change.
There are also a lot of views, that aren't updated according to this renaming.

Best regards

Steffen

Reply:
Thanks @SteffenMahler ,

I will revert this changes.

Reply:
You are welcome.